src/team1/Team1Demo.java

```diff
@@ -5,5 +5,10 @@
 	public static void main(String[] args) {
 		System.out.println("Team 1");
 	}
+	
+	public void subtractNum(int a, int b) {
+		
+		System.out.println(a-b);
+	}
```

Comment:
@ABNERTraining PLEASE PLEASE make sure to format your  CODE before moving to staging. :)